concepts/pattern-matching/introduction.md

```diff
@@ -17,7 +17,7 @@ The use of pattern matching is dominant in assertive, idiomatic Elixir code. You
 
 Remember, matches occur from the right side to the left side.
 
-In the last example if we don't need a variable in a pattern match, we can discard it by referencing `_`. Any variable starting with an `_` is not tracked by the runtime.
+In the last example, if we don't need a variable in a pattern match, we can discard it by referencing `_`. Any variable starting with an `_` is not tracked by the runtime.
```

Comment:
```suggestion
In the last example, if we don't need a variable in a pattern match, we can discard it by referencing `_`.
Any variable starting with an `_` is not tracked by the runtime.
```
Also, changing it to one line per sentence would be great.

Reply:
Done 👌